cron-ci/setup-cron.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -l
```

Comment:
I noticed you are using `-l` so that `$MODULESHOME/init/bash` is sourced because that is the case on HPC's.
When a sudo executes this script, there will be no login shell.

My recommendation is to remove `-l` and do a `source $MODULESHOME/init/sh` wherever modules are being loaded/used/etc.

Reply:
I'll try it out, but I don't think `$MODULESHOME` will even be set. You'll have to `source` the absolute directory like `/apps/contrib/lmod/lmod/init/bash` which is a pain to set for each machine.